src/Domain/FormGroups/FormGroupGateway.php

```diff
@@ -182,7 +182,15 @@ public function getNextFormGroupID($gibbonFormGroupID)
             'gibbonFormGroupID' => $gibbonFormGroupID,
         ]);
     }
-
+  
+    public function getFormGroupDetailsBySchoolYear($gibbonSchoolYearID, $gibbonFormGroupID)
```

Comment:
Overall looking good. Form groups are attached to a school year, so this method can be simplified to only accept a gibbonFormGroupID.
```suggestion
    public function getFormGroupDetailsByID($gibbonFormGroupID)
```